### ISSUE TITLE
No microservice logs in Linux

### ISSUE THREAD

**Issue by codependent** [ErnestOrt/Trampoline#7]
When Trampoline starts the microservices in Windows there is a new cmd window where users can check the execution logs. However in Linux there is no way to see what's going on (logs) with those microservices.

**Comment by ErnestOrt** [comment]
I guess is a general UNIX issue, we should reviewed. Let's included on project tasks and take it as soon as we can do it.

**Comment by ErnestOrt** [comment]
Hi @codependent , 

I've been thinking on it. I would like to remove cmd window on Windows as well... It would be impractical to have 10 windows opened and you being checking in what windows is showing the microservice which contains the error... 

The approach would be to set up log files next to launcher scripts thanks to:
```
logging.file= # Log file name. For instance `myapp.log`
logging.level.*= # Log levels severity mapping. For instance `logging.level.org.springframework=DEBUG`
logging.path= # Location of the log file. For instance `/var/log`
```
and let the user being able to retrieve (or check) them from the instances managing view, next to trace or metrics.

What's your view here?

**Comment by codependent** [comment]
Hi @ErnestOrt, I agreet, having so many open windows is really impractical. Thinking twice about it... what you suggest is already supported through the use of vm arguments. Uses can already specify -Dlogging.file=xxxx when setting them. So I think that just documenting it in the FAQ would be enough.

**Comment by ErnestOrt** [comment]
Thanks! Agree, I just wonder if it would be better to set it for the user... Still not 100% convinced. Anyway, I will add a button to redirect users to logs and add it on FAQ and also remove useless code. This would be a first step anyway 😁 Starting it 🖎

**Comment by ErnestOrt** [comment]
Solved on v3.1 release.
